### PR TITLE
sc-13986 Enabled diabetes option in progress tab with assigned patients

### PR DESCRIPTION
**Pull request by Gyan-Gupta-Rtsl**
**Story card:** [sc-13986](https://app.shortcut.com/simpledotorg/story/13986/enable-diabetes-option-in-progress-tab)

## Because
Enabled the diabetes option with the assigned patients section

## This addresses
Enabled the assigned patients section with values as given in the below screenshots. 

## Test instructions
Enable this flipper flag :diabetes_progress_report_tab to check the functionality.

### CODE REVIEW

**Comment by Gyan-Gupta-Rtsl** [comment]
Without flipper flag:

Screenshot 2024-11-12 at 5 23 20 PM

With Flipper flag:
Screenshot 2024-11-08 at 9 02 51 PM
Screenshot 2024-11-08 at 9 03 13 PM

**Review by danySam — CHANGES_REQUESTED** [review]
Looks ready! 

I’d suggest adding the details of the flipper flag to `⁠/docs/wiki/feature-flags.md`. We capture the details of the feature flags there. 

Also, in your PR’s title and description, you can mention that this only implements assigned patients. At first, I was under the assumption that this would include all the necessary changes for the diabetes progress report.

**Comment by Gyan-Gupta-Rtsl** [comment]
@danySam Can you review this PR.

File: doc/wiki/feature-flags.md [hunk 1]
```diff
@@ -36,4 +36,5 @@ to facility the launch of a new feature, and may reference the Simple team's int
...
 | patients_protected                        | Yes                          | To selectively launch the [patients protected](https://app.shortcut.com/simpledotorg/epic/11405) section on the HTN dashboard.
 | patients_protected_htn_cascade            | Yes                          | To selectively launch the HTN cascade subsection of the [patients protected](https://app.shortcut.com/simpledotorg/epic/11405) feature on the HTN dashboard.
-| diabetes_who_standard_indicator           | Yes                          | When enabled, only FBS and HBA1c measurements are considered for calculating diabetes controlled and uncontrolled rate
\ No newline at end of file
+| diabetes_who_standard_indicator           | Yes                          | When enabled, only FBS and HBA1c measurements are considered for calculating diabetes controlled and uncontrolled rate
+| diabetes_progress_report_tab				| Yes						   | When the progress tab is activated, the diabetes section will be displayed as enabled, followed by a comprehensive listing of all associated key indicators.
```
Comment (danySam) [inline doc/wiki/feature-flags.md:40]:
```suggestion
| diabetes_who_standard_indicator           | Yes                          | When enabled, only FBS and HBA1c measurements are considered for calculating diabetes controlled and uncontrolled rate
| diabetes_progress_report_tab		    | Yes	    		      | When the progress tab is activated, the diabetes section will be displayed as enabled, followed by a comprehensive listing of all associated key indicators.
```

Comment (danySam) [inline doc/wiki/feature-flags.md:40]:
```suggestion
| diabetes_progress_report_tab				| Yes						   | When the progress tab is activated, the diabetes section will be displayed as enabled, followed by a comprehensive listing of all associated key indicators.
```

**Review by danySam — APPROVED** [review]

**Pull request merged**